README.md

```diff
@@ -70,15 +70,15 @@ The command line interface (CLI) allows users to submit their project package de
    phylum analyze <package-lock-file.ext>
    ```
 
- 1. (Optional) View the analysis results in the [Phylum UI](https://app.phylum.io/auth/login)
+1. (Optional) View the analysis results in the [Phylum UI](https://app.phylum.io/auth/login)
```

Comment:
This change also needs to be made in `docs/command_line_tool/quickstart.md`.

And we should probably add the license section there as well.